src/billing/components/BillingPage.tsx

```diff
@@ -52,7 +52,9 @@ function BillingPage() {
         <Page titleTag="Billing">
           <Page.Header fullWidth={false} testID="billing-page--header">
             <Page.Title title="Billing" />
-            <LimitChecker>{!isCancelled && <RateLimitAlert />}</LimitChecker>
+            <LimitChecker>
+              <RateLimitAlert />
```

Comment:
removed isCancelled since it's no longer relevant to the UI (i.e. Cancelled users can't access the app)